### ISSUE TITLE
collection settings naming

### ISSUE THREAD

**Issue by joolswills** [RetroPie/EmulationStation#244]
This was mentioned in another ticket/PR I think - but I would like to rename the collection settings to reduce the wordiness. Any suggestions 

for example (just ideas)

Main settings - from `GAME COLLECTION SETTINGS` to `COLLECTION SETTINGS"

then perhaps we can just have (do we need collections in every line - since the area is all about collections?)

```
AUTOMATIC
CUSTOM
CREATE NEW FROM THEME
CREATE NEW
GROUP UNTHEMED
SORT
```

happy for suggestions. I am thinking I might get these changed before 4.3 as it's minor to change them - as they don't fit well on low resolution screens.

can be documented in the retropie docs for further clarification

**Comment by zigurana** [comment]
Good idea! 
`AUTOMATIC` really means 'from metadata', as that's where the entries are based on, and `CUSTOM` means 'user defined'. For the first, i'd like 
'SELECT FROM METADATA', and for the second 'SELECT FROM CUSTOM'.
With respect to the other entries, I am not sure we really need all these options,but changing that would entail more than just a labeling change, so let's park that.

**Comment by zigurana** [comment]
I appreciate that my suggestions are not what you are looking for, they are even more verbose, but I wanted to explain what the options do.

**Comment by pjft** [comment]
Thanks for starting this. I'm supportive of this kind of changes in general.

My suggestions:

Main settings - from GAME COLLECTION SETTINGS to "GAME COLLECTIONS"

then:

AUTOMATIC
CUSTOM
CREATE FROM THEME
CREATE NEW
GROUP UNTHEMED
SORT

I imagine we would do well in merging the "automatic" and "custom" menu to a single one that just states "Select Active" and lists all collections at once. That is a relic from the past, as I launched Automatic first, and then developed the Custom ones. 

I can take a stab at that at a later stage, unless you think it'd still be on time for 4.3.

Thanks.

**Comment by joolswills** [comment]
I don't mind too much for 4.3 - I had assumed it would just require some strings changes but @zigurana 's comment suggests it's more involved so maybe I can just leave it to you to do when convenient.

**Comment by pjft** [comment]
We can do the string changes for now, and then the remaining ones at our pace. I don't oppose those in any way.

Thanks for paying attention to this.

**Comment by hex007** [comment]
@joolswills Will you be building binaries again? There is a PS patch that I intend to submit in an hour fixing #243 . Let me know so I shall hurry up with this.

**Comment by psyke83** [comment]
@hex007 I don't think it's necessary to rush it? Your patch still doesn't fix the related issue of "press any key" when configuring input broken on the INSTANT profile.

**Comment by hex007** [comment]
@psyke83 Did you check the latest comments on the thread? We can just disable and enable PS to take care of this. But yea this shall be hurried if and only if @joolswills is building binaries again.

**Comment by psyke83** [comment]
I may have misunderstood. You meant for users to manually disable via the menu? I was under the impression that you meant to add PS pause/resume calls to GuiDetectDevice::update (or wherever is best). I assumed you'd update the patch to include something like that.

**Comment by hex007** [comment]
Yes I meant that the code will automatically handle PS pause and resume. 

We -> DEVS
Users -> USERS

**Comment by hex007** [comment]
@psyke83 The 4.3 is already released so :|

**Comment by pjft** [comment]
@zigurana just checking, I believe you are bundling these changes in one of your PRs, is that correct?

**Comment by zigurana** [comment]
That's correct!
